app/assets/stylesheets/arclight/modules/hierarchy.scss

```diff
@@ -36,6 +36,13 @@
     line-height: 1.25;
   }
 
+  .al-document-abstract-or-scope-heading {
```

Comment:
How about `.al-hierarchy-sub-heading`? I found a `.al-show-sub-heading` in _collection_overview.html.erb that seems similar. It's specific to the context, but not the metadata type.